libass/ass.h

```diff
@@ -734,6 +734,24 @@ void ass_process_chunk(ASS_Track *track, const char *data, int size,
  */
 void ass_set_check_readorder(ASS_Track *track, int check_readorder);
 
+/**
+ * \brief Garbage collect events that preceed deadline
+ * \param track track
+ * \param deadline cut-off timestamp in milliseconds.
+*/
+void ass_execute_gc(ASS_Track *track, long long deadline);
```

Comment:
fwiw i liked the previous name, `ass_prune_events`, better (similar change in `\brief`)

Reply:
I went with the _gc prefix to associate it to the other function. I am fine with either.